web/docs/develop-vast/write-documentation.md

```diff
@@ -0,0 +1,72 @@
+# Write Documentation
+
+The VAST documentation resides inside [our main GitHub
+repository](https://github.com/tenzir/vast) in
+[`/web/docs`](https://github.com/tenzir/vast/tree/master/web/docs).
+We use [Docusaurus](https://docusaurus.io/) as site framework.
+
+### Build and View Locally
+
+To view the entire site (including the documentation) locally,
+change to the [`/web`](https://github.com/tenzir/vast/tree/master/web/)
+directory and invoke [yarn](https://yarnpkg.com/):
+
+```bash
+yarn
```

Comment:
We should recommend using `yarn install --frozen-lockfile` over `yarn` so users don't auto-upgrade the lockfile depending on their globally installed packages.

Reply:
Does `yarn` without any arguments really do an auto-upgrade? Every time newer versions were out, I always a get notice in the shell, but the upgrade doesn't happen by default.

Reply:
Per our discussion, I can see now why this is relevant.